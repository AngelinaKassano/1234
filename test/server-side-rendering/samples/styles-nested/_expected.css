div.Atletik-bzh57p{color:red}
div.Atletik-4yw8vx{color:green}