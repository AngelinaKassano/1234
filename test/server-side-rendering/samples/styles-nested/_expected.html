<div class="Atletik-bzh57p">red</div>
<div class="Atletik-4yw8vx">green: foo</div>



<div class="Atletik-4yw8vx">green: bar</div>


